### PR TITLE
drop support for ie9 and ie10; #3148

### PR DESCRIPTION
**Pull request by Bamieh**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -20,4 +20,5 @@ mocha.js
 *_LOCAL_*
 *_REMOTE_*
 docs/_site
+*.sublime-*
```
Comment (boneskull) [inline .gitignore]: please revert; you can add this as a separate PR since it's unrelated

**Comment by Bamieh** [comment]
@boneskull done.

**Comment by boneskull** [comment]
@Bamieh either your or I should rebase this onto `master` (instead of merging `master` into it)

**Comment by Bamieh** [comment]
@boneskull i can do it, but there is an option on the merge button to "rebase and merge". i'd use that for ease

**Pull request merged**